.gitignore

```diff
@@ -3,3 +3,4 @@
 node_modules
 npm-debug.log
 coverage
+jest*
```

Comment:
After running tests locally, I had a folder titled `jest_0` in the repo directory. Cool to gitignore those?

Reply:
I don't see any `jest` directory in the repo, so probably 👍 to add this to the gitignore. I also don't see any `jest*` files in the repo but do you think, just to be safe, `jest*/` is better? I don't think it's a big deal either way, though it might be clearer that `jest*/` takes care of any directories and then `**/jest*` takes care of any files.